soem/ethercattype.h

```diff
@@ -86,9 +86,9 @@ extern "C"
 /** datagram type EtherCAT */
 #define EC_ECATTYPE        0x1000
 /** number of frame buffers per channel (tx, rx1 rx2) */
-#define EC_MAXBUF          16
+#define EC_MAXBUF          256
```

Comment:
Leave ethercattype.h as is, changes here affects all plattforms. I've written an #65 to make it possible to override values.  256 buffers don't make sense on small embedded systems.